website/pages/docs/builders/proxmox.mdx

```diff
@@ -245,6 +245,12 @@ builder.
   - `iso_checksum` (string) - Checksum of the ISO file.
   - `unmount` (bool) - If true, remove the mounted ISO from the template after finishing. Defaults to `false`.
 
+- `http_interface` - (string) - Name of the network interface that Packer gets
```

Comment:
Maybe this should rather go in `common.HTTPConfig`? There's already an option `http_bind_address`, so there should be some care taken on the interaction I guess (what happens if both are specified)

Reply:
That's true, I didn't think of it. I'll put it there👍

Reply:
One more caveat to note here: the http_config is for options that are common to all builders. If http_interface is in this config, it'll get documented in every builder that uses the http_config, even though it is only available to proxmox. 

If you want to leave it in the http_config that's fine especially since I can see wanting to add this feature to other builders, but for now let's tell our docs generator to leave it off the autogenerated docs. You can do this by setting the "undocumented" tag: 

```
HTTPInterface string `mapstructure:"http_interface"
```

It looks like you've already manually documented the option on the proxmox.mdx page, so it should still show up for proxmox even if you pull it from the docs generation.

Reply:
Okay I'll do that :+1: